sql_scripts/9_beers_filtered.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS beer.beers_filtered
+(
+    id INTEGER NOT NULL PRIMARY KEY,
```

Comment:
From what I understand this id field values are actually taken from beer.beers table and they would need to reference values here through FOREIGN KEY